soda_machine.rb

```diff
@@ -1,18 +1,40 @@
+require_relative "soda"
+
 class SodaMachine
   attr_reader :sodas, :cash
 
+  class Soda < SodaMachine
```

Comment:
this makes no sense.  inheritance typically (for now "always") satisfy the "IS_A" relationship.  but saying that a `Soda` IS_A `SodaMachine` doesn't work. why did you inherit here?